cli/cmd/upgrade.go

```diff
@@ -161,23 +79,23 @@ The upgrade can be configured by using the --set, --values, --set-string and --s
 A full list of configurable values can be found at https://www.github.com/linkerd/linkerd2/tree/main/charts/linkerd2/README.md
 `,
 
-		Example: `  # Default upgrade - also removes linkerd resources that no longer exist in the current version
+		Example: `  # Upgrade CRDs first
+  linkerd upgrade --crds | kubectl apply --prune --prune-whitelist=apiextensions.k8s.io/v1/customresourcedefinitions
+
+  # Then upgrade the controle-plane - also removes linkerd resources that no longer exist in the current version
```

Comment:
This reads a little weird right now

```suggestion
  # Then upgrade the controle-plane which also removes linkerd resources that no longer exist in the current version
```